strongbox-data-service/src/main/resources/db/changelog/v1.0.0/v1.0.0.8__AuthorizationConfig.xml

```diff
@@ -8,6 +8,8 @@
 
         <o:dropProperty className="AuthorizationConfig" propertyName="roles" />
 
+        <o:delete tableName="Roles"/>
```

Comment:
@fuss86 you probably need to add same statement after dropping class properties. Seems that OrientDB is not assumed to drop data within dropping a property.

Reply:
@sbespalov thanks for this tip. Why have you used _o_ namespace prefix here ? _delete_ element comes from liquibase core as far as I see. Will it work prefixed with _o_ namespace ?

Reply:
And how can I use it to remove all values from the property it they were not wrapped by additional element (like `Repository#groupRepositories`) ?

Reply:
Maybe we should try to use standard liquibase _update_ data change element